core/src/main/java/com/netflix/conductor/core/execution/tasks/Inline.java

```diff
@@ -0,0 +1,113 @@
+/*
+ *  Copyright 2021 Netflix, Inc.
+ *  <p>
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ *  the License. You may obtain a copy of the License at
+ *  <p>
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  <p>
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ *  an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ *  specific language governing permissions and limitations under the License.
+ */
+package com.netflix.conductor.core.execution.tasks;
+
+import static com.netflix.conductor.common.metadata.tasks.TaskType.TASK_TYPE_INLINE;
+
+import java.util.Map;
+
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+import com.netflix.conductor.common.metadata.tasks.Task;
+import com.netflix.conductor.common.run.Workflow;
+import com.netflix.conductor.core.exception.TerminateWorkflowException;
+import com.netflix.conductor.core.execution.WorkflowExecutor;
+import com.netflix.conductor.core.execution.evaluators.Evaluator;
+
+/**
+ * @author X-Ultra
+ * <p>
+ * Task that enables execute inline script at workflow execution, For example,
+ * <pre>
+ * ...
+ * {
+ *  "tasks": [
+ *      {
+ *          "name": "INLINE",
+ *          "taskReferenceName": "inline_test",
+ *          "type": "INLINE",
+ *          "inputParameters": {
+ *              "input": "${workflow.input}",
+ *              "evaluatorType": "javascript"
+ *              "expression": "if ($.input.a==1){return {testvalue: true}} else{return {testvalue: false} }"
+ *          }
+ *      }
+ *  ]
+ * }
+ * ...
+ * </pre>
+ * then to use task output, e.g.  <code>script_test.output.testvalue</code>
+ */
+@Component(TASK_TYPE_INLINE)
+public class Inline extends WorkflowSystemTask {
```

Comment:
Please mark LAMBDA as deprecated with a comment indicating this as the replacement

Reply:
Done.